build/win64/Dockerfile

```diff
@@ -6,8 +6,9 @@ ENV AR=x86_64-w64-mingw32-ar
 
 RUN apt update && apt dist-upgrade -y && \
     apt install -y make git cmake g++-mingw-w64-x86-64 \
-    binutils-mingw-w64-x86-64 wget \
-    apt-get install autoconf
+    binutils-mingw-w64-x86-64 wget
+    
+RUN sudo apt-get install autoconf
```

Comment:
The `\` before the newline says "just ignore the fact that there's a newline here.

So you just had to do like:

```
binutils-mingw-w64-x86-64 wget \
autoconf
```

Reply:
I see.  Watching me try to figure out how to get this to work should be entertaining.